specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/networkVirtualAppliance.json

```diff
@@ -285,6 +285,67 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkVirtualAppliances/{networkVirtualApplianceName}/restart": {
+      "post": {
+        "tags": [
+          "NetworkVirtualAppliances"
+        ],
+        "operationId": "NetworkVirtualAppliances_Restart",
+        "description": "Restarts one or more VMs belonging to the specified Network Virtual Appliance.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "networkVirtualApplianceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of Network Virtual Appliance."
```

Comment:
Please handle linter error:

The resource name parameter 'networkVirtualApplianceName' should be defined with a 'pattern' restriction.

Reply:
I spoke with the ARM team. I will be submitting a suppression for this error tomorrow